playbooks/deploy_bugziller.yml

```diff
@@ -1,20 +1,3 @@
----
-- hosts: myrmicinae.rht.gluster.org
+- hosts: bugziller.rht.gluster.org
   roles:
-  - role: guest_virt_install
-    name: bugziller.rht.gluster.org
-    volgroup: myrmicinae
-    disk_size: 10
-    mem_size: 1024
-    num_cpus: 1
-    distribution: Centos
-    version: 7
-    bridge: virbr1
-    sshkey: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
-    network:
-      bootproto: static
-      ip: 8.43.85.178/24
-      gateway: 8.43.85.254
-      nameservers:
-      - 8.8.8.8
-      - 8.8.4.4
+  - role: bugzilla
```

Comment:
Shouldn't it be called "bugziller", rather than "bugzilla" ? It doesn't install bugzilla.